sql/clickhouse/ClickHouseLexer.g4

```diff
@@ -0,0 +1,296 @@
+
+// $antlr-format alignColons trailing, alignLabels true, alignLexerCommands true, alignSemicolons ownLine, alignTrailers true
+// $antlr-format alignTrailingComments true, allowShortBlocksOnASingleLine true, allowShortRulesOnASingleLine true, columnLimit 150
+// $antlr-format maxEmptyLinesToKeep 1, minEmptyLines 0, reflowComments false, singleLineOverrulesHangingColon true, useTab false
+
+lexer grammar ClickHouseLexer;
```

Comment:
Could you please use [`caseInsensitive`](https://github.com/antlr/antlr4/blob/dev/doc/options.md#caseinsensitive) option and replace fragments with string literals? See [SQL](https://github.com/antlr/grammars-v4/tree/master/sql) grammars as an example.

Reply:
You mean That will be a big work and make little sense

Reply:
Thank you for your advice. But replacing all fragments will make a big change to the g4 file in clickhouse source code and  makes little sense to the lexer result. Is it necessary to do this format change?

Reply:
It's not necessary, but desired since it makes grammar more clear and other grammars in the repository uses this option.

Reply:
Thanks. I will consider optimizing it when I have the chance.